configure

```diff
@@ -2187,14 +2187,35 @@ check_higgstools_libs() {
             return 1
         fi
     else
+        local filename
+        filename=$(basename -- "$found_lib")
+        local extension="${filename##*.}"
```

Comment:
I'm wondering if this is bashism. I have to check if this works with `dash`.

Reply:
Sorry, I meant the statement `"${filename##*.}"`. I'm not sure if this is a bashism. Maybe it is not, I'm not sure.

Reply:
I've checked that it works with `dash`. Also, according to [this](https://unix.stackexchange.com/a/277754), we are already checking for bashisms.

Reply:
Excellent, thanks!